docs/_managedk8s/FAQ/automatic_dns/index.en.md

```diff
@@ -0,0 +1,166 @@
+        name: external-dns-viewer
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: external-dns
+      subjects:
+        - kind: ServiceAccount
+          name: external-dns
+          namespace: external-dns
+      ```
+    * Secret:
+      ```yaml
+      apiVersion: v1
+      kind: Secret
+      type: Opaque
+      metadata:
+        name: designate-openstack-credentials
+        namespace: external-dns
+      stringData:
+        OS_AUTH_URL: https://identity.optimist.innovo.cloud/v3
+        OS_REGION_NAME: fra
+        OS_AUTH_TYPE: v3applicationcredential
+        OS_APPLICATION_CREDENTIAL_ID: <appcred_id>
+        OS_APPLICATION_CREDENTIAL_SECRET: <appcred_secret>
+      ```
+    * Deployment:
+      ```yaml
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: external-dns
+        namespace: external-dns
+      spec:
+        selector:
+          matchLabels:
+            app: external-dns
+        strategy:
+          type: Recreate
+        template:
+          metadata:
+            labels:
+              app: external-dns
+          spec:
+            serviceAccountName: external-dns
+            containers:
+              - args:
+                  - --source=service
+                  - --source=ingress
+                  - --registry=txt
+                  - --provider=designate
+                  - --domain-filter=example.gec.io
+                  - --txt-owner-id=<owner-id>
+                  - --log-level=debug
+                envFrom:
+                  - secretRef:
+                      name: designate-openstack-credentials
+                image: registry.k8s.io/external-dns/external-dns:latest
+                imagePullPolicy: IfNotPresent
+                name: external-dns
+
+      ```
+
+1. **Annotate** the service or ingress with `external-dns.alpha.kubernetes.io/hostname: my-app.example.gec.io` e.g.
+    ```yaml
+    ---
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: my-app
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: nginx
+      namespace: my-app
+    spec:
+      selector:
+        matchLabels:
+          app: nginx
+      template:
+        metadata:
+          labels:
+            app: nginx
+        spec:
+          containers:
+          - image: nginx
+            name: nginx
+            ports:
+            - containerPort: 80
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: nginx
+      namespace: my-app
+      annotations:
+        external-dns.alpha.kubernetes.io/hostname: my-app.example.gec.io
+    spec:
+      selector:
+        app: nginx
+      type: LoadBalancer
+      ports:
+        - protocol: TCP
+          port: 80
+          targetPort: 80
+    ```
+1. Make the dns record will be looked up correct:
+    ```sh
+    $ openstack recordset list example.gec.io.
+    $ dig my-app.example.gec.io @dns1.ddns.innovo.cloud.
```

Comment:
make clear that they need to update their main domain to delegate to designate: https://github.com/gecio/gecio.github.io/blob/6fb320cfcda9fd18a9097c90e4a382f0c4ab886e/docs/_gks/k8sapplications/externaldnsanddesignate/index.en.md?plain=1#L25-L33